Gems/AtomLyIntegration/CommonFeatures/Code/Source/CoreLights/DirectionalLightComponentConfig.cpp

```diff
@@ -132,7 +132,7 @@ namespace AZ
 
         bool DirectionalLightComponentConfig::IsEsmDisabled() const
         {
-            return !m_shadowEnabled || (m_shadowFilterMethod != ShadowFilterMethod::Esm && m_shadowFilterMethod != ShadowFilterMethod::EsmPcf);
+            return !m_shadowEnabled || !(m_shadowFilterMethod == ShadowFilterMethod::Esm || m_shadowFilterMethod == ShadowFilterMethod::EsmPcf);
```

Comment:
good ol' De Morgan

Reply:
This change is more correct from compiler point of view, need to check which error that was and it was not related to De Morgan per se.